apps/tx-builder/src/hooks/useElementHeight/useElementHeight.tsx

```diff
@@ -0,0 +1,17 @@
+import { RefObject, useEffect, useState } from 'react';
+
+const useElementHeight = (elementRef: RefObject<HTMLElement>) => {
```

Comment:
The purpose of this custom hook is to return the height of an element (used in the TransactionDetails Component to implement the Show more/less button)

![Captura de pantalla 2022-02-24 a las 18 54 53](https://user-images.githubusercontent.com/26763673/155580241-9d45aaa3-a217-4640-87da-c98f9edbde32.png)